x-pack/plugins/observability_solution/observability/public/plugin.ts

```diff
@@ -421,6 +436,7 @@ export class Plugin
                   sortKey: 100,
                   entries: [
                     ...overviewLink,
+                    ...inventoryLink,
```

Comment:
since this is POC, should this be behind feature flag?

Reply:
the plugin itself is disabled by default, I don't want this to leak to the UI anywhere to regular users so I haven't used an advanced setting or anything